### PR TITLE
Base accounts page creation login

### PR DESCRIPTION
**Pull request by David0817**
add create a new account
update back to login
i18n
#29

### CODE REVIEW

File: src/language/en-US.json [hunk 1]
```diff
@@ -772,10 +772,13 @@
   "login_account": "Login",
   "login": "Login",
   "create_a_new_account": "Create a new account",
-  "Access_My_Account": "Access My Account",
-  "Do_not_have_Account": "You don't have an account yet?",
+  "go_to_login": "Go to login",
+  "Access_My_Account": "Choice One Method Access Symbol",
```
Comment (evias) [inline src/language/en-US.json]: `Access my account`,  please do not modify existing translations.

File: src/language/en-US.json [hunk 2]
```diff
@@ -772,10 +772,13 @@
   "login_account": "Login",
   "login": "Login",
   "create_a_new_account": "Create a new account",
-  "Access_My_Account": "Access My Account",
-  "Do_not_have_Account": "You don't have an account yet?",
```
Comment (evias) [inline src/language/en-US.json]: Why did you remove this? This sentence is necessary.

Comment (NemTechCN) [reply src/language/en-US.json]: https://github.com/nemfoundation/symbol-desktop-wallet/issues/29 
how to work ?

Comment (David0817) [reply src/language/en-US.json]: Thank you for your correction，I will review my code more carefully。

File: src/language/en-US.json [hunk 3]
```diff
@@ -772,10 +772,14 @@
   "login_account": "Login",
   "login": "Login",
   "create_a_new_account": "Create a new account",
+  "go_to_login": "Go to login",
   "Access_My_Account": "Access My Account",
   "Do_not_have_Account": "You don't have an account yet?",
+  "Existing_Account": "Existing account",
```
Comment (evias) [inline src/language/en-US.json]: `Do you have an account already?`

**Comment by evias** [comment]
there is too much _empty space_ in the screen

Screenshot 2020-03-17 at 11 44 17 AM

File: src/language/en-US.json (hunk 3 above)
Comment (David0817) [reply src/language/en-US.json]: image
like this?

Comment (evias) [reply src/language/en-US.json]: 1. remove the comma `,` after the question sign.

2. Replace `Go to login` with `Click here to login now.`

Comment (David0817) [reply src/language/en-US.json]: O(∩_∩)O

**Comment by evias** [comment]
It is now possible to click the back button even when there is no wallets known. Please fix.

**Comment by David0817** [comment]
this is a guide page,The more concise the page, the more focused it will be on each option, i think.

**Comment by evias** [comment]
rebased and merged in master
